src/styles/components/pages/landingPages/homePage.scss

```diff
@@ -1,14 +1,28 @@
 // Home Page
 .home-page-wrapper {
   height: 100vh;
+  margin-bottom: 2rem;
+
+  header {
+    position: fixed;
+    width: 100%;
+  }
+
+  .nav-menu {
```

Comment:
This breaks the nav (hides the first element on mobile view), we'll go over a better fix at the merge meeting